llvm/test/tools/llvm-cas/validation.test

```diff
@@ -0,0 +1,9 @@
+RUN: rm -rf %t
+RUN: mkdir %t
+
+RUN: llvm-cas --cas %t/cas --ingest %S/Inputs > %t/cas.id
+RUN: llvm-cas --cas %t/cas --validate
```

Comment:
Some additional sanity tests
* -check-hash
* remove data pool now that you are validating that too
* remove standalone file (except I can't think of an easy way to generate a large enough file, so fine to skip this if you can't think of a way either)

Reply:
Generate a large file is easy. Removing the standalone file is not easy. I don't think a wildcard expansion that relies on the shell works too well in unit test.

Removing data pool is already covered.

Reply:
> Removing the standalone file is not easy. I don't think a wildcard expansion that relies on the shell works too well in unit test.

Good point.  I guess you could copy the data+index to a different directory to "remove" all the standalone files.  I'll leave it to your judgment if it's worth it.